Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -7,6 +7,17 @@
 import Foundation
 import WebKit
 
+internal protocol WebRUMEventContextProviding {
+    var context: RUMContext? { get }
+}
+
+internal class WebRUMEventContextProvider: WebRUMEventContextProviding {
+    var context: RUMContext? {
+        // TODO: RUMM-1786 implement web event context provider
+        return nil
+    }
+}
```

Comment:
here is what android has: [WebRumEventContextProvider](https://github.com/DataDog/dd-sdk-android/blob/feature/hybrid-application/dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/WebRumEventContextProvider.kt)

this new type is probably not needed, i guess we can achieve this by injecting `Global.rum.contextProvider` if RUM is enabled.

Reply:
Yes, we already have this thing in `dd-sdk-ios` and it is abstracted by `RUMContextProvider` interface. I don't see reason for duplicating it by another `WebRUMEventContextProvider` or `WebRUMEventContextProviding` constructs, do we?